labgrid/util/marker.py

```diff
@@ -1,6 +1,178 @@
 import random
 import string
+import re
+import time
+
+from ..protocol import CommandProcessProtocol
+from ..step import step
+from ..util import Timeout, OptionalTimeout
+from pexpect import EOF, TIMEOUT, spawn
```

Comment:
This is a third party import. Move it up, so it is above the local package imports (standard imports first, then third party, then local package).

Reply:
Done